src/main/scala/is/hail/experimental/package.scala

```diff
@@ -0,0 +1,34 @@
+package is.hail
+
+import is.hail.stats._
+
+package object experimental {
+
+  def findMaxAC(af: Double, an: Int, ci: Double = .95): Int = {
+   if (af == 0) {
+      0
+    } else {
+      val quantile_limit = ci // ci for one-sided, 1-(1-ci)/2 for two-sided
+      val max_ac = qpois(quantile_limit, an * af)
+      max_ac
```

Comment:
The val isn't necessary here because you immediately return it, but keep it if you like it for documentation.